content/docs/optimizing-performance.md

```diff
@@ -1,93 +1,91 @@
 
 ### Single-File Builds {#single-file-builds}
 
-We offer production-ready versions of React and React DOM as single files:
+Ofrecemos versiones listas para producción de React y React DOM como un archivo:
 
 ```html
 <script src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-Remember that only React files ending with `.production.min.js` are suitable for production.
+Recuerda que solo los archivos de React que terminan con `production.min.js` son apropiados para producción.
 
 ### Brunch {#brunch}
 
-For the most efficient Brunch production build, install the [`uglify-js-brunch`](https://github.com/brunch/uglify-js-brunch) plugin:
+Para la *build* de producción Brunch más eficiente, instala el plugin [`uglify-js-brunch`](https://github.com/brunch/uglify-js-brunch):
 
 ```
-# If you use npm
+# Si usas npm
 npm install --save-dev uglify-js-brunch
 
-# If you use Yarn
+# Si usas Yarn
 yarn add --dev uglify-js-brunch
 ```
 
-Then, to create a production build, add the `-p` flag to the `build` command:
+Entonces, para crear una *build* de producción, agrega la bandera `-p` al comando `build`:
 
 ```
 brunch build -p
 ```
-
-Remember that you only need to do this for production builds. You shouldn't pass the `-p` flag or apply this plugin in development, because it will hide useful React warnings and make the builds much slower.
+Recuerda que solo necesitas hacer esto para las *build* de producción. Tú no deberías pasar la bandera `-p` o aplicar el plugin en desarrollo, por que ocultará las advertencias de React y hará las *builds* mucho mas lentas.
 
 ### Browserify {#browserify}
 
-For the most efficient Browserify production build, install a few plugins:
+Para la *build* de producción con Browserify más eficiente, instala algunos plugins:
 
 ```
-# If you use npm
+# Si usas npm
 npm install --save-dev envify uglify-js uglifyify 
 
-# If you use Yarn
+# Si usas Yarn
 yarn add --dev envify uglify-js uglifyify 
 ```
+Para crear una *build* de producción, asegúrate de agregar estas transformaciones **(El orden es importante)**:
 
-To create a production build, make sure that you add these transforms **(the order matters)**:
-
-* The [`envify`](https://github.com/hughsk/envify) transform ensures the right build environment is set. Make it global (`-g`).
-* The [`uglifyify`](https://github.com/hughsk/uglifyify) transform removes development imports. Make it global too (`-g`).
-* Finally, the resulting bundle is piped to [`uglify-js`](https://github.com/mishoo/UglifyJS2) for mangling ([read why](https://github.com/hughsk/uglifyify#motivationusage)).
+* La transformación [`envify`](https://github.com/hughsk/envify) asegura que el ambiente de la *build* sea correcto. Hazlo lo global (`-g`).
+* La transformación [`uglifyify`](https://github.com/hughsk/uglifyify) remueve los import de desarollo. Hazlo global tambien (`-g`).
```

Comment:
```suggestion
* La transformación [`uglifyify`](https://github.com/hughsk/uglifyify) remueve los *imports* de desarollo. Hazlo global también (`-g`).
```
I think we should use the same format for 'imports' as we do for 'build' and others.